OloEngine/src/OloEngine/Renderer/Debug/CommandPacketDebugger.cpp

```
@@ -1,834 +1,1104 @@
-        if (!m_DrawKeyStats.m_DepthDistribution.empty())
+        // Render state for commands that have one
+        if (cmd.GetCommandType() == CommandType::DrawMesh ||
+            cmd.GetCommandType() == CommandType::DrawMeshInstanced ||
+            cmd.GetCommandType() == CommandType::DrawSkybox ||
+            cmd.GetCommandType() == CommandType::DrawInfiniteGrid ||
+            cmd.GetCommandType() == CommandType::DrawQuad)
         {
-            ImGui::Text("Depth Distribution (%zu depths):", m_DrawKeyStats.m_DepthDistribution.size());
-            RenderDrawKeyHistogram(m_DrawKeyStats.m_DepthDistribution, "Depth Distribution");
-        }
+            const PODRenderState* state = nullptr;
```

Clang (14.0.6)

[warning] variable 'state' is not initialized

(cppcoreguidelines-init-variables)